subworkflows/local/build_references/main.nf

```diff
@@ -18,6 +18,8 @@ include { CTATSPLICING_PREPGENOMELIB }      from '../../../modules/local/ctatspl
     IMPORT NF-CORE MODULES/SUBWORKFLOWS
 ========================================================================================
 */
+
+include { FUSIONCATCHER_BUILD }             from '../../../modules/nf-core/fusioncatcher/build/main'
```

Comment:
should be download

Reply:
The download include is at the top of the local modules